clients.json

```diff
@@ -1085,7 +1085,15 @@
     "description": "A Redis client written with the Akka IO package introduced in Akka 2.2.",
     "authors": ["chrisdinn"]
   },
-
+  {
+    "name": "redisclient",
+    "language": "Rust",
+    "repository": "https://github.com/ltoddy/redis-rs",
+    "description": "Redis client for Rust.",
+    "authors": ["ltoddy"],
```

Comment:
This is supposed to have Twitter handles, not GitHub's. If you don't have a Twitter handle, just remove the authors key entirely.